scripts/build/Platform/Windows/build_config.json

```diff
@@ -213,19 +213,38 @@
       "ASSET_PROCESSOR_PLATFORMS": "pc,server"
     }
   },
-  "periodic_test_profile_vs2019_pipe": {
+  "awsi_test_profile_vs2019_pipe": {
     "TAGS": [
       "nightly-incremental",
       "nightly-clean"
     ],
     "steps": [
       "awsi_deployment",
-      "periodic_test_profile_vs2019"
+      "awsi_test_profile_vs2019"
     ]
   },
+  "awsi_test_profile_vs2019": {
+    "TAGS": [
+        "weekly-build-metrics"
+    ],
+    "COMMAND": "build_test_windows.cmd",
+    "PARAMETERS": {
+      "CONFIGURATION": "profile",
+      "OUTPUT_DIRECTORY": "build\\windows_vs2019",
+      "CMAKE_OPTIONS": "-G \"Visual Studio 16 2019\" -DCMAKE_SYSTEM_VERSION=10.0 -DLY_UNITY_BUILD=TRUE",
+      "CMAKE_LY_PROJECTS": "AutomatedTesting",
+      "CMAKE_TARGET": "TEST_SUITE_awsi",
+      "CMAKE_NATIVE_BUILD_ARGS": "/m /nologo",
+      "CTEST_OPTIONS": "-L \"(SUITE_awsi)\" -T Test",
```

Comment:
In some places, we have exclusion filters that I think need to be fixed, for example:
`scripts\build\Platform\Linux\build_config.json`, `test_profile` and `test_profile_nounity` are doing:
```
-E Gem::EMotionFX.Editor.Tests -LE SUITE_sandbox -L FRAMEWORK_googletest
```
That excludes the EMotionFX.Editor.Tests, and the sandbox suite, for the the googletest framework
If you have `awsi` tests in googletest, it will run in linux under that job
Looks like that is the only place, I think you just need to replace: `-LE SUITE_sandbox` with `-LE (SUITE_sandbox|SUITE_awsi)`

Reply:
Just updated the PR to address this comment. Thanks.